app/src/main/res/values/strings.xml

```diff
@@ -399,4 +399,11 @@
     <string name="metric_newest">\"SUSI, what are the newest skills?\"</string>
     <string name="metrics_top_games">\"SUSI, what are your top games?\"</string>
     <string name="message_no_skills_found">No skills found.</string>
+
+    <string name="report_skill">Report Skill</string>
+    <string name="report_send">Send Report</string>
+    <string name="report_error">Coudn\'t report the skill</string>
```

Comment:
'Error reporting skill' or may be check some other standard apps for a more standard string.